replication-controllers/sync-gateway.yaml

```diff
@@ -22,10 +22,8 @@ spec:
           image: couchbase/sync-gateway
           command:
             - sync_gateway
-            - https://raw.githubusercontent.com/couchbase/kubernetes/master/config/sync-gateway.config
+            - { "log": ["*"], "databases": { "db": { "server": "http://$COUCHBASE_SERVER_SERVICE_HOST:8091", "bucket": "default", "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } } } } }
```

Comment:
I don't think this will work, because AFAIK that needs to be in a file or on a URL.  Let me know if you know otherwise from testing it.